chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -278,10 +278,10 @@ sync
 <% else -%>
   <% if @target_platform_version == "11.3" -%>
       <package>suse-sle11-openstack-cloud-release</package>
```

Comment:
Strictly speaking, we'd need the supportutils package here too (but I'd be equally fine removing 11.3 from there)